scheduler/src/cook/mesos.clj

```diff
@@ -255,6 +258,15 @@
                                                                               :pending-jobs-atom mesos-pending-jobs-atom
                                                                               :trigger-chan rebalancer-trigger-chan
                                                                               :view-incubating-offers view-incubating-offers})
+                                    (when (seq optimizer-config)
+                                      (cook.mesos.optimizer/start-optimizer-cycles! (fn get-queue []
+                                                                                      ;; TODO Use filters scheduler uses
```

Comment:
I don't understand this TODO

Reply:
The matcher filters the ranked jobs to construct the considerable list (https://github.com/twosigma/Cook/blob/master/scheduler/src/cook/mesos/scheduler.clj#L726). For example, handling quota correctly.

It is probably ok for a first pass to ignore that but is worth adding in a separate PR.

Reply:
Makes sense. Is it worth adding some of this context to the TODO comment?